pkg/securitypolicy/securitypolicyenforcer.go

```diff
@@ -483,17 +585,117 @@ func equalForOverlay(a1 []string, a2 []string) bool {
 	return true
 }
 
-func possibleIndicesForID(containerID string, mapping map[int]map[string]struct{}) []int {
-	var possibles []int
-	for index, ids := range mapping {
+func (pe *StandardSecurityPolicyEnforcer) possibleIndicesForID(containerID string) []int {
```

Comment:
could we either make these helper functions not be on the `StandardSecurityPolicyEnforcer` object and just pass through what the function needs OR make a comment/make it clear that these helper functions must be called while holding the mutex lock already?

Reply:
added a comment.